src/ResourceManagement/Compute/Domain/IVirtualMachineScaleSetVM.cs

```diff
@@ -174,7 +174,7 @@ public interface IVirtualMachineScaleSetVM :
         /// <summary>
         /// Gets virtual machine instance size.
         /// </summary>
-        Models.VirtualMachineSizeTypes Size { get; }
+        string Size { get; }
```

Comment:
[`VirtualMachineSizeTypes`](https://github.com/Azure/azure-libraries-for-net/blob/master/src/ResourceManagement/Compute/Generated/Models/VirtualMachineSizeTypes.cs) is heavily used type, we should continue to support it, may be as handwritten enum type. This type is used in VM create/update flow so it impact major use cases.